### DISCUSSION TITLE
Accessing GitHub Copilot Workspace

### DISCUSSION THREAD

**Discussion by anastasiabusygina** [community/community#87460] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Hello!

We are keen to explore the potential of GitHub Copilot Workspace for our organization's projects. However, we are uncertain about the process to gain access to this tool. Could you please provide information on the following:

What is the specific procedure or point of entry to access GitHub Copilot Workspace?
Is there a formal application, registration process, or any other specific action required to request access to Copilot Workspace?
Your guidance on these queries will greatly aid us in understanding the steps to access this innovative tool.

Thank you for your assistance.

Best regards,
Anastasia,
Chainhackers

**Comment by mishmanners** [comment, marked as answer]
Thanks for reaching out. You can find out more about GitHub Copilot Workspaces here: https://githubnext.com/projects/copilot-workspace/

If this is something you're interested to try out, please reach out to your GitHub Account Manager. If you don't have one yet, please contact the GitHub Enterprise Sales team via: https://github.com/enterprise/contact. You can also reach out directly to the GitHub Copilot Workspaces team via: next@github.com

**Comment by Qassemkareem** [comment]
i have smae problem when i try to access to workspace , it was message error apear "Access denied
You don't have access to Copilot Workspace. If you think this is an error, try [signing in again](https://copilot-workspace.githubnext.com/auth/signout)." 
how to fix this it
